openerp/openupgrade/doc/source/modules80-90.rst

```diff
@@ -50,7 +50,7 @@ Status :
 +-----------------------------------+-----------------------------------+
 |auth_crypt                         | Nothing to do                     |
 +-----------------------------------+-----------------------------------+
-|auth_ldap                          |                                   |
+|auth_ldap                          | No change                         |
```

Comment:
You're right, although there's a connotation: if there's no analysis file, then "No change" is the correct, but we have used "Nothing to do" for both things, so better to unify it.